jaeger-core/src/main/java/com/uber/jaeger/Configuration.java

```diff
@@ -263,56 +261,79 @@ public void setStatsFactory(StatsFactory statsFactory) {
     this.statsFactory = statsFactory;
   }
 
+  public Configuration reporterConfiguration(ReporterConfiguration reporterConfig) {
```

Comment:
nit: naming not consistent with setStatsFactory or with builders. Any reason not to converge on `withXyz` naming?

nit2: Configuration suffix can probably be dropped